app/src/main/res/values/styles.xml

```diff
@@ -259,11 +259,16 @@
         <item name="android:layout_height">match_parent</item>
     </style>
 
+    <style name="BlueCursor" parent="ThemeOverlay.MaterialComponents.TextInputEditText.OutlinedBox">
```

Comment:
Can we rename this style from `BlueCursor` to `TextInputEditTextTheme`? Using `Blue` in the style name will force you to change it (and all the usages) if you change the property inside it. Better to decouple concrete values from the style name, that way we can change them under the hood without big refactors.

Reply:
Hi @cmonfortep 
I am just reading this. Sounds good to me. I'll change the name here